internal/repositories/user_test.go

```diff
@@ -39,11 +39,12 @@ func (s *UserSuite) SetupTest() {
 
 	s.users = make(map[int]*ent.User)
 	for i := 1; i <= 12; i++ {
+		name := fmt.Sprintf("user%d", i)
 		s.users[i] = &ent.User{
 			Login:    fmt.Sprintf("user_%d", i),
 			Email:    fmt.Sprintf("user_%d@mail.com", i),
 			Password: "password",
-			Name:     fmt.Sprintf("user%d", i),
+			Name:     &name,
```

Comment:
Why creating a separate variable instead of assigning value directly in User struct?